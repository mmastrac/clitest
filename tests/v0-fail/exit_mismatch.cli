#!/usr/bin/env clitest --v0

$ exit 1
%EXIT 2
