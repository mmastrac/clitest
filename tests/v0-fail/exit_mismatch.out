Running <root>/exit_mismatch.cli ...
exit 1
---
(no output)
---
❌ FAIL: exit status: 1
 → exit 1

<root>/exit_mismatch.cli FAILED
Error: exit status: 1

